system/integrations/checkmk/api-reference.rst

```diff
@@ -0,0 +1,173 @@
+   :alt: Checkmk article body
+   :align: center
+
+These details come from the fields listed below,
+which correspond to parameters provided by Checkmk (``$NOTIFY_*``).
+Technically, they can be replaced with whatever you want,
+but it’s not recommended.
+
+**Required fields are marked with an asterisk (\*).**
+
+event_id\*
+   A unique ID for the system event. (``$NOTIFY_SERVICEPROBLEMID`` / ``$NOTIFY_HOSTPROBLEMID``)
+
+host\*
+   The hostname of the system that the event originated from. (``$NOTIFY_HOSTNAME``)
+
+   Used to determine if a new event belongs to an existing ticket.
+   Also used in the subject line of the resulting article (“<host> is <state>”).
+
+service
+   The name of the service that the event originated from. (``$NOTIFY_SERVICEDESC``)
+
+   Used to determine if a new event belongs to an existing ticket.
+
+   Displayed as ``-`` when omitted.
+
+state\*
+   The current state of the service or host in question. (``$NOTIFY_SERVICESTATE`` / ``$NOTIFY_HOSTSTATE``)
+
+   Used to detect when a ticket should be auto-closed (*i.e.,* on ``OK``/``UP``).
+   Also used in the subject line of the resulting article (“<host> is <state>”).
+
+text
+   The output of the process that triggered the event. (``$NOTIFY_SERVICEOUTPUT`` / ``$NOTIFY_HOSTOUTPUT``)
+
+   Displayed as ``-`` when omitted.
+
+Ticket Attributes
+-----------------
+
+.. figure:: /images/system/integrations/checkmk/finding-object-names.png
+   :alt: The Object Manager attribute panel displays built-in and custom attribute names.
+   :align: center
+   :width: 80%
+
+   Find a complete list of ticket attributes in the Object Manager.
+
+Pass ticket attributes to change the settings of the ticket
+that will be created for this event.
+(Note that these attributes will be ignored
+if the event belongs to an existing ticket.)
+
+Why would you want to do this?
+Maybe you have only one IT guy,
+and all system monitoring issues should be automatically assigned to him.
+Or, maybe you’re creating multiple notification rules
+so that database outages take higher priority
+than disk space warnings.
+
+In most cases, you’ll probably want to set one of the following:
+
+* group
+* owner
+* state
+* priority
+
+but in practice, you can set almost any attribute,
+including :doc:`custom ones you created through the Object Manager </system/objects>`.
+
+.. note:: 🙅 The following attributes are **not customizable**:
+
+   * title
+   * id
+   * ticket number
+   * customer
+   * created_by_id
+   * updated_by_id
+
+How do I know what values I can set?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+group / state / priority
+   .. figure:: /images/system/integrations/checkmk/ticket-attribute-names.png
+      :alt: See possible values for certain attributes in the ticket pane.
+      :scale: 50%
+      :align: center
+
+   Refer to the dropdown menus in the ticket pane:
+
+   .. code:: bash
+
+      -F "group=Users"
+      -F "state=closed"
```

Comment:
Attention on state!
This is a default checkmk attribute that does **not set the ticket state**!

My original commit contained this:
```
State
   You can set a specific state id to create the ticket with another state than it usually would (new).
   
      * Reference way ``state_id=2``
```

This is one of the reasons why I decided to provide the user with two options: named and id based references. State is an exception where only setting via ID is possible.

Reply:
This is a really tricky point. I'm glad you pointed this out, because I 100% didn't catch it the first time around. I think this deserves a note; working on rewriting it now.

Reply:
This is addressed in the latest commit. Leaving this conversation open so that you can a) resolve it if you're okay with it, or b) continue the discussion if you're not.